js/app.js

```diff
@@ -0,0 +1,100 @@
+//What is the difference between interpolation and concatenation?
+// Interpolation uses a variable to represent a value.
+//Concatenation combines two 'strings' together.
+
+//Give an example of each.
+// Interpolation:
+//// const verb = 'believe'
+// const noun = 'cheese'
+// console.log("Please " + verb + " the " + noun + ".")
+//
+//Concatenation:
+//const cityName = 'Astoria'
+//const weather = 'rainy'
+//console.log(cityName + " is " + weather + " this time of year.");
+//**ARE ALL CONCATENATIONS INTERPOLATIONS BUT NOT ALL INTERPOLATIONS ARE CONCATENATIONS??**
+
+//What does the acronym DRY stand for?
+//Don't Repeat Yourself
+
+//Why should we pay attention to it?
+//Helps utilize best practices in programming. DRY makes our code easier to read by other programmers
+//and helps us avoid bulky code and unessecary repetition.
+
+//What programming tools have we learned to write DRY code?
+//Thematically we discussed understaning the specific task at hand before rendering the code.
+//Specifially, we learned to use loops to avoid code repitition.
+
+//Section 2: Boolean Expressions
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a<b);
+console.log(c>d);
+console.log('Name'=== 'Name');
+console.log(a<b<c);
+console.log(a<a<d);
+console.log(e==='Kevin');
+console.log(48!=='48');
+
+
+//Section 3: While loops
+//while (true) {
+//console.log('Do not run this loop');
+//}
+//This loop is an infinite loop. The loop does not contain a "code block" and
+//there is no control expression.
+
+//Section 3: Inifinite Loop II
+//Is this loop inifinite? Why or why not?
+//const runProgram = true;
+// while (runProgram) {
+	// console.log('Do not run this loop');
+	// runProgram = false;
+// }
+//This loop is not infinite. It meets the while loop criteria using:
+//1. initial condition 2. while condition & 3. repeating expression/operation??
+
+let letters = "A";
+let i = 0;
+
+while (i < 20) {
+	letters += "A";
+	i++;
+}
+
+console.log(letters);
+
+//+= assigns the result of the addition to the variable 'A'
+//instructed to print letter A as many times as the loop.
+//Yes, but console printed A twenty one times rather than the twenty I thought.
+
+//Section 4: For loops
+//A for loop performs the same operation as a while loop.
+//But what are the key differences, if any? What are the similarities?
+
+//For loop 1
+//for (let i = 0; i <= 999; i++) {
+//  console.log(i);
+//}
+
+//Both loops use a defined condition for running the code block.
+//However, For loops set a variable before the loop starts
+//and increases a value each time the code block has been executed.
```

Comment:
it sounds like you might be conflating the two a little.  for loops determine how many times the loop will run in their control structure, including increasing (or decreasing, or changing in some other way, or whatever) the value each time.  an expression that controls a while loop may be set before the loop itself, or depend on other data in the application.  also its entirely possible for a while loop to increase a value each time the code runs (or not) by doing the increment inside the code block (or not).  lmk if you have questions about this.